README.md

```diff
@@ -123,6 +132,13 @@ pip install super-gradients
 
 ## What's New
 
+__________________________________________________________________________________________________________
+Version 3.4.0 (November 6, 2023)
+
+* [YoloNAS-Pose](YOLONAS-POSE.md) model released - a new frontier in pose estimation
+* Added option to export a recipe to a single YAML file or to a standalone train.py file 
+* Other bugfixes & minor improvements. Full release notes available [here](https://github.com/Deci-AI/super-gradients/releases/tag/3.4.0)
+
 __________________________________________________________________________________________________________
 Version 3.1.3 (July 19, 2023)
```

Comment:
It's a little weird to show `3.4.0` and then `3.1.3` and `3.1.1`, there is no clear logic and it may look a bit unprofessional.
Maybe we should just remove `3.1.3` and `3.1.1`, and try to stick to adding the release notes in every the next versions ?

Reply:
I'm not a big fan of putting release notes in the README tbh.
Usually a dedicated file is more clean way to keep those, however I don't see why need this at all. Releases page is pretty good at providing clean picture what is new.
If we look at major libraries like pytorch - they don't update their README with each new version.